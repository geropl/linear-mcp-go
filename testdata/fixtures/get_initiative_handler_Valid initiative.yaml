---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 220
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tquery GetInitiative($id: String!) {\n\t\t\tinitiative(id: $id) {\n\t\t\t\tid\n\t\t\t\tname\n\t\t\t\tdescription\n\t\t\t\turl\n\t\t\t}\n\t\t}\n\t","variables":{"id":"3bb752a7-897e-4240-9306-01e48872fab3"}}'
        form: {}
        headers:
            Content-Type:
                - application/json
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"data":{"initiative":{"id":"3bb752a7-897e-4240-9306-01e48872fab3","name":"Created Test Initiative","description":null,"url":"https://linear.app/linear-mcp-go-test/initiative/created-test-initiative-7ed59af889f6"}}}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"d8-hd6//06VmE0Pm1n9UOQfuP2hyLQ"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 0s
