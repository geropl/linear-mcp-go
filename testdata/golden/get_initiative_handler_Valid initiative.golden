err: ""
output: |
    Initiative: Created Test Initiative
      ID: 3bb752a7-897e-4240-9306-01e48872fab3
      URL: https://linear.app/linear-mcp-go-test/initiative/created-test-initiative-7ed59af889f6
